.clang-format

```diff
@@ -219,7 +219,14 @@ PenaltyIndentedWhitespace: 0
 PenaltyReturnTypeOnItsOwnLine: 200
 PointerAlignment: Middle
 PPIndentWidth:   -1
-QualifierAlignment: Leave
+QualifierAlignment: Custom
+QualifierOrder:
+  - static
+  - inline
+  - friend
```

Comment:
`friend` is the rarest qualifier by far. I would prefer it be the first.

Reply:
Easy fix will do for next time.